src/libs/Navigation/Navigation.ts

```diff
@@ -97,6 +97,10 @@ function getActiveRoute(): string {
         return '';
     }
 
+    if (currentRoute?.path) {
+        return currentRoute.path;
+    }
+
```

Comment:
Should we remove the rest of the logic and only keep this one?

Reply:
@dukenv0307 I just found that `currentRoute.path` is only available for the first navigated route. We may need to use the rootFromState as a fallback here


https://github.com/Expensify/App/assets/16493223/d2b7cbd6-1564-4774-a3dd-e709ff9c4ef9